2022-01-14 12:00:02,303 INFO    MainThread:338455 [wandb_setup.py:_flush():71] setting env: {}
2022-01-14 12:00:02,303 INFO    MainThread:338455 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 12:00:02,303 INFO    MainThread:338455 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 12:00:02,303 INFO    MainThread:338455 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_120002-3vt4pnt9/logs/debug.log
2022-01-14 12:00:02,303 INFO    MainThread:338455 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_120002-3vt4pnt9/logs/debug-internal.log
2022-01-14 12:00:02,303 INFO    MainThread:338455 [wandb_init.py:init():404] calling init triggers
2022-01-14 12:00:02,303 INFO    MainThread:338455 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-14 12:00:02,303 INFO    MainThread:338455 [wandb_init.py:init():460] starting backend
2022-01-14 12:00:02,303 INFO    MainThread:338455 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-14 12:00:02,309 INFO    MainThread:338455 [backend.py:ensure_launched():216] starting backend process...
2022-01-14 12:00:02,313 INFO    MainThread:338455 [backend.py:ensure_launched():221] started backend process with pid: 338517
2022-01-14 12:00:02,313 INFO    MainThread:338455 [wandb_init.py:init():469] backend started and connected
2022-01-14 12:00:02,317 INFO    MainThread:338455 [wandb_init.py:init():533] updated telemetry
2022-01-14 12:00:02,326 INFO    MainThread:338455 [wandb_init.py:init():563] communicating current version
2022-01-14 12:00:02,693 INFO    MainThread:338455 [wandb_init.py:init():568] got version response 
2022-01-14 12:00:02,693 INFO    MainThread:338455 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-14 12:00:03,400 INFO    MainThread:338455 [wandb_init.py:init():606] starting run threads in backend
2022-01-14 12:00:03,457 INFO    MainThread:338455 [wandb_run.py:_console_start():1810] atexit reg
2022-01-14 12:00:03,457 INFO    MainThread:338455 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-14 12:00:03,457 INFO    MainThread:338455 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-14 12:00:03,458 INFO    MainThread:338455 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-14 12:00:03,458 INFO    MainThread:338455 [wandb_init.py:init():633] run started, returning control to user process
2022-01-14 12:00:03,652 INFO    MainThread:338455 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-14 12:00:03,653 INFO    MainThread:338455 [wandb_run.py:_restore():1752] restore
2022-01-14 12:00:05,972 INFO    MainThread:338455 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 12:00:06,143 INFO    MainThread:338455 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 12:00:06,403 INFO    MainThread:338455 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 12:00:06,506 INFO    MainThread:338455 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 4094
}

2022-01-14 12:00:06,608 INFO    MainThread:338455 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4094
  total_bytes: 4094
}

2022-01-14 12:00:06,711 INFO    MainThread:338455 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4094
  total_bytes: 4094
}

2022-01-14 12:00:06,815 INFO    MainThread:338455 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4094
  total_bytes: 4094
}

2022-01-14 12:00:06,919 INFO    MainThread:338455 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4094
  total_bytes: 4094
}

2022-01-14 12:00:07,020 INFO    MainThread:338455 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4094
  total_bytes: 4094
}

2022-01-14 12:00:07,539 INFO    MainThread:338455 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4094
  total_bytes: 4094
}

2022-01-14 12:00:07,906 INFO    MainThread:338455 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4094
  total_bytes: 4094
}
local_info {
}

2022-01-14 12:00:08,954 INFO    MainThread:338455 [wandb_run.py:_append_files():2180] logging synced files
